content/en/modes/dashboard.md

```diff
@@ -0,0 +1,134 @@
+
+## Advanced Customization
+
+`lem-dashboard` is designed to be customizable. If you wish to make more changes than `set-default-dashboard` allows, you can make your own dashboard layout.
+
+A dashboard is simply a list of `dashboard-item` instances. Here's a list of available items:
+
+- `dashboard-splash`: Randomly displays one of `SPLASH-TEXTS`.
+- `dashboard-url`: Creates link/button with `DISPLAY-TEXT` that opens `URL` externally.
+- `dashboard-working-dir`: Prints current working directory.
+- `dashboard-footer-message`: Randomly displays one of the passed-in `MESSAGES`.
+- `dashboard-command`: Creates a link/button with `DISPLAY-TEXT` that executes `ACTION-COMMAND` when clicked/activated.
+- `dashboard-recent-projects`: Displays a list of recent projects, limited to the last `PROJECT-COUNT`.
+- `dashboard-recent-files`: Displays a list of recent files, limited to the last `FILE-COUNT`.
+
+All you need to do is call `set-dashboard` and pass it a list of `dashboard-item` instances:
+
+```lisp
+(in-package :lem-dashboard)
+
+(set-dashboard (list (make-instance 'dashboard-splash
+                                    :item-attribute 'document-metadata-attribute
+                                    :splash-texts '("Welcome!" "Second splash message!")
+                                    :top-margin 4
+                                    :bottom-margin 2)
+                     (make-instance 'dashboard-working-dir)
+                     (make-instance 'dashboard-recent-files 
+                                    :file-count 5
+                                    :bottom-margin 1)
+                     (make-instance 'dashboard-command
+                                    :display-text " New Lisp Scratch Buffer (l)"
+                                    :action-command 'lem-lisp-mode/internal:lisp-scratch 
+                                    :item-attribute 'document-header2-attribute
+                                    :bottom-margin 2)))
+```
+
+The base class, `dashboard-item`, has the following slots:
+
+- `item-attribute`: The attribute used when drawing the item
+- `top-margin`: The amount of vertical space (lines) to apply before the item.
+- `bottom-margin`: The amount of vertical space (lines) to apply after the item.
+- `action`: Function to execute when `RET` is pressed over this item. Most of the existing `dashboard-item` implementations have good default behavior here.
+
+### Creating your own `dashboard-item`
+
+Creating a custom `dashboard-item` is straightforward; just derive the class and implement the `draw-dashboard-item` method.
+
+You are responsible for the actual insertion of text inside of `draw-dashboard-item`. There is a convenience function, `create-centered-string`, which you can use when inserting text.
+
+As an example, let's make a `dashboard-item` that fetches a random fact from an HTTP API and prints it:
+
+```lisp
+(defclass dashboard-random-fact (dashboard-item)
```

Comment:
you'll need to show which package we are in.

Reply:
Done, thanks!